weave/trace/weave_client.py

```diff
@@ -1325,6 +1391,55 @@ def fail_call(self, call: Call, exception: BaseException) -> None:
         """Fail a call with an exception. This is a convenience method for finish_call."""
         return self.finish_call(call, exception=exception)
 
+    @trace_sentry.global_trace_sentry.watch()
+    def get_call_descendants(
+        self,
+        calls: Call | WeaveObject | CallId | Iterable[Call | WeaveObject | CallId],
+        depth: int | None = None,
+        limit: int | None = None,
+    ) -> Iterable[Call | WeaveObject]:
+        """Get all descendant calls of given list of calls or call IDs.
+
+        Args:
+            calls: A single call/object/id or iterable of them
+            depth: Maximum depth of descendants to fetch
+            limit: Maximum number of descendants to fetch
+
+        Returns:
+            An iterable of descendant calls
+        """
+        # Convert input to a list of items we can process
+        if isinstance(calls, (Call, WeaveObject, CallId)):
+            items_to_process = [calls]
+        elif isinstance(calls, Iterable):
+            items_to_process = list(calls)
```

Comment:
I don't think you need to call `list` here

Reply:
`weave/trace/weave_client.py:1416: error: Incompatible types in assignment (expression has type "WeaveObject | Iterable[Call | WeaveObject | str]", variable has type "list[Call | WeaveObject | str]")  [assignment]`

Reply:
ill fix by defining items_to_process explicitly as an iterable

Reply:
nothing I do seems to work....

Reply:
seems like a type-checker mistake -- you already do an isinstance check for `WeaveObject` above

I won't block on this, it's probably fine